### ISSUE TITLE
Microsoft Edge Add-on

### ISSUE THREAD

**Issue by ThomasIAm** [Bloemendaal/Windesheim-ELO#27]
Ik ben overgegaan van Google Chrome naar Microsoft Edge sinds Microsoft eindelijk naar Chromium over is. Het zou wel mooi zijn als de plugin ook in de Microsoft Edge store zou komen.

**Comment by Bloemendaal** [comment]
Ik dacht dat je vanuit edge gewoon Chrome plugins kon installeren? Ik zal maandag even voor je kijken. Evt. vind ik het ook prima als je hem zelf naar de edge store upload.

**Comment by ThomasIAm** [comment]
Ja, je kunt zeker Chrome plugins installeren, maar het is wat gebruiksvriendelijker als het in de Microsoft store staat. Ik zal even kijken of ik hem in de store kan krijgen.

**Comment by ThomasIAm** [comment]
![image](https://user-images.githubusercontent.com/10333951/82765051-be26c300-9e13-11ea-9308-67af052bb65a.png)
Kom een aantal dingen tegen zoals dit ^. Het is vrij simpel om de plugin te uploaden, wil je het zelf doen of vertel je me wat ik daar in moet vullen?

**Comment by Bloemendaal** [comment]
De scripts die je hebt toegevoegd zijn niet van toepassing voor Chrome toch? In dat geval lijkt het me het best om een `Edge` branch te onderhouden met die extra bestanden. Verder lijkt dat screenshot me best wel een probleempje, je kan namelijk niet je schoolaccount doorgeven aan Microsoft maar zonder account is het enige dat deze plugin doet je redirecten. Dependencies staan in de readme.md.

**Comment by ThomasIAm** [comment]
Zoals je misschien hebt gemerkt, heb ik even geen tijd om hier wat mee te doen. Je zou altijd zelf de plugin kunnen uploaden.
